docker-compose/control-compose.yml

```diff
@@ -2,11 +2,11 @@
 version: '3.2'
 services:
   data:
-    image: ns1inc/privatedns_data:${TAG:-1.1.1}
+    image: nsone/standalone_data:${TAG:-2.1.0}
```

Comment:
Are we no longer using the `ns1inc` dockerhub org and `privatedns` containers?  All images in this compose are pointing to `nsone` org/`standalone` containers

Reply:
Good call.

Reply:
Should've been changed when brought over here.